### DISCUSSION TITLE
Building SWDAP

### DISCUSSION THREAD

**Discussion by simudvarac** [ARMmbed/DAPLink#807] [General]
Hi,

I would need a new FW for our SWDAP clone programmer.
Building the project results in a lot of binaries for different platforms but the needed SWDAP is missing.
I see the swdap.c is present in the boards but that is all.
Can somebody please advise what steps I need to do to build the swdap board.
Thank you very much in advance!
Best
Miki

**Comment by mathias-arm** [comment]
You can try the `lpc11u35_tiny_if_crc.bin` or `lpc11u35_if_crc.bin`.

**Reply by simudvarac** [reply]
Thank you, this worked for us!
